specification/deviceprovisioningservices/resource-manager/readme.md

```diff
@@ -105,6 +114,19 @@ swagger-to-sdk:
       - node sdkauto_afterscript.js deviceprovisioningservices/resource-manager
 ```
 
+## Suppression
+``` yaml
+directive:
+  - suppress: SECRET_PROPERTY
+    from: 
+      - Microsoft.Devices/preview/2021-02-01-preview/iotdps.json
+    where: 
+      - $.definitions.SharedAccessSignatureAuthorizationRule[AccessRightsDescription].properties.primaryKey
+      - $.definitions.SharedAccessSignatureAuthorizationRule[AccessRightsDescription].properties.secondaryKey
+      - $.definitions.VerificationCodeResponse.properties.verificationCode
```

Comment:
The correct property name is $.definitions.VerificationCodeResponse.properties.properties.properties.verificationCode

Reply:
Fixed